clients/fides-js/src/lib/ui-utils.ts

```diff
@@ -0,0 +1,6 @@
+export const blockPageScrolling = () => {
+  document.body.classList.add("fides-no-scroll");
+};
+export const unblockPageScrolling = () => {
+  document.body.classList.remove("fides-no-scroll");
```

Comment:
if for some reason this class isn't found, what will happen?

Reply:
Just google it and tested it on my browser. It does nothing, it doesn't throw an error.